### ISSUE TITLE
OpenAIR: Add additional command TP to provide transponder code

### ISSUE THREAD

**Issue by reskume** [naviter/seeyou_file_formats#8]
For certain airspaces, it would make sense to have an optional TP command that provides a transponder code, e.g. for a TMZ.

**Comment by lordfolken** [comment]
i´d  second such a field.

**Comment by reskume** [comment]
With the next big feature update for OpenAIP (which will happen in the next weeks), we add support for adding transponder settings to each airspace. Each settings consists of `code`, `primary` flag and a `remarks` field. For an airspace with multiple settings, the first "primary" transponder setting will be exported into a TP command which will contain the `code` value.

**Comment by akolarck** [comment]
@reskume Have you already rolled out this feature? If there's already an example of it, I'd like to add it to our backlog.

**Comment by reskume** [comment]
Yep, it's available since some time now but I forgot to update the docs of our parser. Please find the docs here: [[TokenIntrospectionResult](https://github.com/openAIP/openaip-openair-parser?tab=readme-ov-file#reads-extended-openair-airspace-definitions-with-extendedformat-true)](https://github.com/openAIP/openaip-openair-parser?tab=readme-ov-file#reads-extended-openair-airspace-definitions-with-extendedformat-true)

The parser is *only* used to ingest OpenAIR into OpenAIP. So, basically, you can see what OpenAIP can handle depending on what is documented in the parser repo.
